File: skills/astra-toolkit/clients/typescript/examples/admin-find-database-regions.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

const client = new DataAPIClient(process.env.ASTRA_DB_APPLICATION_TOKEN!);

const admin = client.admin();

(async function () {
  const regions = await admin.findAvailableRegions();

  console.log(regions);
})();

File: skills/astra-toolkit/clients/csharp/examples/admin-find-database-regions.cs
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    var client = new DataAPIClient(System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN"));

    var admin = client.GetAstraDatabasesAdmin();

    var regions = await admin.FindAvailableRegionsAsync();
  }
}
